### PR TITLE
Add fetching stream info

### PR DESCRIPTION
**Pull request by Douile**
This commit adds the functionality to fetch additional info from the
twitch "streams" API. This is then used to display streamers streaming
non-infosec stuff (filtered by the game name) as offline and also
display the listed language.

The updateStreamStatus() function was also completley rewritten to make
the additional displayed info easier to implement.

I am adding this as a draft as:
- I have yet to test markdown generation as it's a bit of a pain to setup eventsub
- Table headers need to be changed
- Might also want to display language in a more pretty way (e.g. emoji)

Anyways let me know what you think :heart: 
btw this is aiming to fix https://github.com/infosecstreams/infosecstreams.github.io/issues/59

### CODE REVIEW

**Comment by Douile** [comment]
This also might be using the wrong API, I used https://dev.twitch.tv/docs/api/reference#get-streams instead of https://dev.twitch.tv/docs/api/reference#get-channel-information :man_facepalming:

File: StreamStatus.go [hunk 1]
```diff
@@ -352,14 +406,38 @@ func main() {
...
 		auth:          auth,
 		indexFilePath: filePath,
 		repoPath:      repoPath,
 		url:           repoUrl,
+    client:         client,
 	}
 
+  // Test auth token works
+  // log.Print(repo.fetchStreamInfo("alh4zr3d"))
+
```
Comment (goproslowyo) [inline StreamStatus.go:440]:
```suggestion

```

File: StreamStatus.go [hunk 2]
```diff
@@ -303,8 +315,12 @@ func (s *StreamersRepo) eventsubStatus(w http.ResponseWriter, r *http.Request) {
 		log.Printf("got offline event for: %s\n", offlineEvent.BroadcasterUserName)
 		w.WriteHeader(200)
 		w.Write([]byte("ok"))
+
+
```
Comment (goproslowyo) [inline StreamStatus.go:319]:
```suggestion
```

File: StreamStatus.go [hunk 3]
```diff
@@ -332,6 +358,33 @@ func (s *StreamersRepo) eventsubStatus(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *StreamersRepo) fetchStreamInfo(user_login string) (*helix.Stream, error) {
```
Comment (goproslowyo) [inline StreamStatus.go:361]:
```suggestion
func (s *StreamersRepo) fetchStreamInfo(user_id string) (*helix.Stream, error) {
```

File: StreamStatus.go [hunk 4]
```diff
@@ -332,6 +358,33 @@ func (s *StreamersRepo) eventsubStatus(w http.ResponseWriter, r *http.Request) {
...
+  streams, err := s.client.GetStreams(&helix.StreamsParams{
+    UserLogins: []string{user_login},
```
Comment (goproslowyo) [inline StreamStatus.go:363]:
```suggestion
    UserIDs: []string{user_id},
```

**Comment by Douile** [comment]
Maybe this should insert `&nbsp;` in the empty columns, I forgot to add that.

**Pull request merged**